source/docs/troubleshooting.md

```diff
@@ -235,7 +235,31 @@ FATAL Something's wrong. Maybe you can find the solution here: http://hexo.io/do
 Template render error: (unknown path)
 ```
 
-One possible reason is that there are some unrecognizable words in your file, e.g. invisible zero width characters.
+Possible cause:
+- There are some unrecognizable words in your file, e.g. invisible zero width characters.
+- Incorrect use or limitation of [tag plugin](/docs/tag-plugins).
+  * Tag plugin is not enclosed with `{% endplugin_name %}`
```

Comment:
Note not all the tag plugin is required to be ended with `endplugin_name`, and the wording here might cause confusion.